templates/bot/ts/workflow/src/internal/initialize.ts

```diff
@@ -0,0 +1,23 @@
+import { HelloWorldCardActionHandler } from "../cardActions/helloworldActionHandler";
+import { HelloWorldCommandHandler } from "../commands/helloworldCommandHandler";
+import { ConversationBot } from "@microsoft/teamsfx";
+
+// Create the command bot and register the command handlers for your app.
+// You can also use the commandBot.command.registerCommands to register other commands
+// if you don't want to register all of them in the constructor
+export const commandBot = new ConversationBot({
+  // The bot id and password to create BotFrameworkAdapter.
+  // See https://aka.ms/about-bot-adapter to learn more about adapters.
+  adapterConfig: {
+    appId: process.env.BOT_ID,
+    appPassword: process.env.BOT_PASSWORD,
+  },
+  command: {
```

Comment:
Probably to add a comment here why command is needed here - to initiate the workflow?